core/string/ustring.h

```diff
@@ -520,6 +520,8 @@ class [[nodiscard]] String {
 	String get_basename() const;
 	String path_join(const String &p_path) const;
 	char32_t unicode_at(int p_idx) const;
+	bool has_extension(const char *p_ext) const;
+	bool has_extension(const String &p_ext) const { return has_extension(p_ext.get_data()); }
```

Comment:
This doesn't work, `get_data` returns `char32_t`, so the other method would have to be `bool has_extension(const char32_t *p_ext) const;

Reply:
It is used in several places